resources/styles/global/typography.less

```diff
@@ -1,92 +1,27 @@
-@googleFontApi: "//fonts.googleapis.com/css?family=";
-@fontPath: "/fonts/";
-
-@font-face {
-  font-family: 'Architects Daughter';
-  src: local('Architects Daughter'),
-       local('ArchitectsDaughter'),
-       // url("@{googleFontApi}Architects+Daughter"),
-       url("@{fontPath}ArchitectsDaughter.ttf") format('truetype');
-}
-
-@font-face {
-    font-family: "Lato";
-    src: local('Lato-Light'),
-         // url("@{googleFontApi}Lato:300"),
-         url("@{fontPath}Lato-Light.ttf") format('truetype');
-    font-weight: 300;
-}
-
-@font-face {
-    font-family: "Lato";
-    src: local('Lato-LightItalic'),
-         // url("@{googleFontApi}Lato:300italic"),
-         url("@{fontPath}Lato-LightItalic.ttf") format('truetype');
-    font-weight: 300;
-    font-style: italic, oblique;
-}
-
-@font-face {
-    font-family: "Lato";
-    src: local('Lato-Regular'),
-         // url("@{googleFontApi}Lato:400"),
-         url("@{fontPath}Lato-Regular.ttf") format('truetype');
-    font-weight: 400;
-}
-
-@font-face {
-    font-family: "Lato";
-    src: local('Lato-RegularItalic'),
-         // url("@{googleFontApi}Lato:400italic"),
-         url("@{fontPath}Lato-RegularItalic.ttf") format('truetype');
-    font-weight: 400;
-    font-style: italic, oblique;
-}
-
-@font-face {
-    font-family: "Lato";
-    src: local('Lato-Bold'),
-         // url("@{googleFontApi}Lato:700"),
-         url("@{fontPath}Lato-Bold.ttf") format('truetype');
-    font-weight: 700;
-}
-
-@font-face {
-    font-family: "Lato";
-    src: local('Lato-Black'),
-         // url("@{googleFontApi}Lato:900"),
-         url("@{fontPath}Lato-Black.ttf") format('truetype');
-    font-weight: 900;
-}
-
-@font-face {
-  font-family: 'Merriweather';
-  src: local('Merriweather Light'),
-       local('Merriweather-Light'),
-       // url("@{googleFontApi}Merriweather:300"),
-       url("@{fontPath}Merriweather-Light.ttf") format('truetype');
-  font-weight: 300;
+.tutor-font-face(@name, @filename, @weight, @style:normal) {
+  @font-face {
+    font-family: @name;
+    src: local(@name),
+    url('fonts/@{filename}.woff2') format('woff2'),
+    url('fonts/@{filename}.woff') format('woff');
+    font-weight: @weight;
+    font-style: @style;
+  }
 }
 
```

Comment:
I think @dak wrote these initially and noted that it was difficult to remove the copy/pasta.

1. Is there a reason for including the weight and style here?
2. can people that use this mixin use a token (no Quotes) instead of a string?
  - that way we **could** have LESS do some compile-time-checks